src/Crons/RakutenItemUpdateCron.php

```diff
@@ -45,5 +45,6 @@ public function handle(ItemUpdateService $itemUpdateService)
                 'line' => $throwable->getLine(),
             ]);
         }
+        $itemUpdateService->client->closeAll();
```

Comment:
Why do you not simply close the connections at the end of `generateContent()`?

Reply:
If you tell me the connections have not to be closed in case of an exception, I can only close after generating content

Reply:
You are welcome to add additional try catches =D But please define a new type of exception which could be especially catched just for this use case. I would recommend placing the catch besides the finishing functions at ~ItemUpdateService.php:322

Reply:
why would I define an extra exception? I want the connections closed in all cases